scripts/linters/pylint_extensions.py

```diff
@@ -830,7 +830,8 @@ def check_docstring_structure(self, node):
         if node.doc:
             docstring = node.doc.splitlines()
             # Check for space after """ in docstring.
-            if docstring[0][0] == b' ':
+            # The first check below helps avoiding empty docstring situation.
+            if not docstring[0] == '' and docstring[0][0] == b' ':
```

Comment:
```
def something():
    """
    """
```

In the above situation the value of `docstring` is ['', '       '] so the docstring[0][0] is out of range.

Reply:
Done.